src/Sentry.NLog/SentryNLogOptions.cs

```diff
@@ -42,31 +42,42 @@ public int ShutdownTimeoutSeconds
         /// </summary>
         public bool IgnoreEventsWithNoException { get; set; } = false;
 
+        /// <summary>
+        /// Determines whether context-level properties will be sent to sentry as additional data. Defaults to <see langword="true" />.
+        /// </summary>
+        /// <seealso cref="SendEventPropertiesAsTags" />
+        public bool SendContextPropertiesAsData { get; set; } = true;
+
+        /// <summary>
+        /// Determines whether event-level properties will be sent to sentry as additional data. Defaults to <see langword="true" />.
+        /// </summary>
+        /// <seealso cref="SendEventPropertiesAsTags" />
+        public bool SendEventPropertiesAsData { get; set; } = true;
+
         /// <summary>
         /// Determines whether event properties will be sent to sentry as Tags or not. Defaults to <see langword="false" />.
         /// </summary>
         /// <remarks>
         /// If set to <see langword="false" />, event properties will still be sent as additional data unless
-        /// <see cref="SendLogEventInfoPropertiesAsData" /> is set to <see langword="false" />.
+        /// <see cref="SendContextPropertiesAsData" /> is set to <see langword="false" />.
         /// </remarks>
-        /// <seealso cref="SendLogEventInfoPropertiesAsData" />
-        public bool SendLogEventInfoPropertiesAsTags { get; set; } = false;
+        /// <seealso cref="SendContextPropertiesAsData" />
+        public bool SendEventPropertiesAsTags { get; set; } = false;
 
         /// <summary>
-        /// Determines whether event properties will be sent to sentry as additional data. Defaults to <see langword="true" />.
+        /// Determines whether event properties will be sent to sentry as Tags or not. Defaults to <see langword="true" />.
         /// </summary>
-        /// <seealso cref="SendLogEventInfoPropertiesAsTags" />
-        public bool SendLogEventInfoPropertiesAsData { get; set; } = true;
+        public bool SendContextPropertiesAsTags { get; set; } = true;
 
         /// <summary>
-        /// Determines whether event properties will be sent to sentry as Tags or not. Defaults to <see langword="true" />.
+        /// Determines whether or not to include event-level data as data in breadcrumbs for future errors. Defaults to <see langword="false"/>.
         /// </summary>
-        public bool SendContextPropertiesAsTags { get; set; } = true;
+        public bool IncludeEventDataOnBreadcrumbs { get; set; } = false;
 
         /// <summary>
         /// Any additional tags to apply to each logged message.
         /// </summary>
-        public IDictionary<string, Layout> Tags { get; } = new Dictionary<string, Layout>();
+        public IList<TargetPropertyWithContext> Tags { get; } = new List<TargetPropertyWithContext>();
```

Comment:
Remember to add attribute `[ArrayParameter(typeof(TargetPropertyWithContext), "tag")]` for this property.